internal/controllers/topology/cluster/patches/variables/variables.go

```diff
@@ -64,6 +68,18 @@ type ClusterTopologyBuiltins struct {
 	Class string `json:"class,omitempty"`
 }
 
+// ClusterNetworkBuiltins represents builtin cluster network variables.
+type ClusterNetworkBuiltins struct {
+	// ServiceDomain is the domain name for services.
+	ServiceDomain *string `json:"serviceDomain,omitempty"`
+	// ServicesCIDRs is network ranges from which Service networks are allocated.
```

Comment:
```suggestion
	// ServicesCIDRs are network ranges from which Service VIPs are allocated.
```
I think this should be really service VIPs. As far as I'm aware there are no service networks.

As far as I know:
* kube-controller-manager will use this range to hand out new service VIPs
* the PodsCIDRs are used to create new per-node "pod networks" == CIDRs (at least with some default Kubernetes CNI IPAM implementation)

Reply:
I've copied these comments from the API types which I think it's best to follow to reduce confusion. If we want to discuss the comments on the API types lets move it to a new issue.

Reply:
Okay, kind of agree, but I'm probably missing something. Those are the comments on the API types:
```go
	// The network ranges from which service VIPs are allocated.
	// +optional
	Services *NetworkRanges `json:"services,omitempty"`

	// The network ranges from which Pod networks are allocated.
	// +optional
	Pods *NetworkRanges `json:"pods,omitempty"`
```

Reply:
Sorry - you're dead right. I'll update.

Reply:
Happens :)